scalability_and_performance/index.adoc

```diff
@@ -0,0 +1,63 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="index"]
+= {product-title} Scalability and Performance overview
+include::_attributes/common-attributes.adoc[]
+:context: index
+
+// ifndef::openshift-dedicated,openshift-rosa[]
+{product-title} offers best practices and utilities to help you optimize the performance and scale of your {product-title} clusters. This book in the documentation provides information on recommended performance and scalability practices, reference design specifications, optimization and low latency tuning.
+
+To get support on these topics, refer to the section of the documentation entitled xref:../support/getting-support.adoc#getting-support[Getting support]
+
+// endif::openshift-dedicated,openshift-rosa[]
+
+[NOTE]
+====
+Operators described in this section have release cycles that are separate from the core {product-title} release cycles. The release schedule for the Node Observability Operator can be found here link:access.redhat.com/support/policy/updates/openshift_operators[Openshift Operators]
```

Comment:
This is both too broad and too specific - "Operators described in this section" implies ALL Operators, but you single out the Node Observability Operator in the next sentence. Try this: 

"Some Performance and Scalability Operators have release cycles that are independent from {product-title} release cycles. For more information see, link:...[OpenShift Operator Life Cycles]"

This language is consistent, directs users to the appropriate information, and doesn't single out any Operators.

Reply:
and included the end-of-sentence period.